### ISSUE TITLE
HybridSampler throwing an error

### ISSUE THREAD

**Issue by devraj89** [MaxHalford/pytorch-resample#1]
Hi

Thanks for the wonderful code.
Can you please recheck you HybridSampler implementation?
I could not run the examples and the benchmark code from anywhere especially for the HybridSampler.
The other parts of the code runs absolutely fine.

Thanks
Devraj

**Comment by MaxHalford** [comment]
Does this happen when you install the library? What error are you getting?

**Comment by devraj89** [comment]
Hi I do not have access to pip and am not allowed to install any libraries so I copied your codes in the jupyter notebook and the run the codes which u have shown.

I could run successfully the under-sampler and over-sampler but the hybrid-sampler gave error.

It's actually throwing a value error.

I used the below code 

`>>> sample = pytorch_resample.HybridSampler(
...     dataset=dataset,
...     desired_dist={0: .33, 1: .33, 2: .33},
...     sampling_rate=.5,  # use 50% of the dataset
...     seed=42
... )
`
